src/components/Signup/ExistingPodForm.jsx

```diff
@@ -33,37 +37,28 @@ const ExistingPodForm = () => {
   };
 
   return (
-    <Card
-      sx={{
-        padding: '8px',
-        margin: '8px',
-        boxShadow: '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)'
-      }}
-    >
-      <CardHeader title="Register with Existing Pod" />
-      <form onSubmit={loginHandler} style={formRowStyle}>
-        {/* TODO: Consider changing this to Autocomplete like the NavBar and SignInModal have */}
-        <TextField
-          sx={{ margin: '8px' }}
-          id="existing-pod-provider"
-          aria-label="Pod Provider"
-          label="Pod Provider"
-          variant="outlined"
-          onChange={(e) => setOidcIssuer(e.target.value)}
-          required
-        />
-        <br />
-        <Button
-          variant="contained"
-          color="primary"
-          size="large"
-          aria-label="Login to Pod Provider"
-          type="submit"
-        >
-          Login to Pod Provider
-        </Button>
-      </form>
-    </Card>
+    <Box component="form" onSubmit={loginHandler} sx={formRowStyle} textAlign="center">
+      {/* TODO: Consider changing this to Autocomplete like the NavBar and SignInModal have */}
```

Comment:
Was wondering if we could simply use the current one on the navbar. I believe that's what the PodRegistrationForm is using currently when creating new pods?

Reply:
Sounds good! I'll open a new improvement issue for it.